deploy/docker/clickhouse-setup/otel-collector-config.yaml

```diff
@@ -109,3 +127,8 @@ service:
     metrics/spanmetrics:
       receivers: [otlp/spanmetrics]
       exporters: [prometheus]
+    # uncomment the pipeline below for logs
```

Comment:
Why comment this? If I am sending logs over OTLP just like traces and metrics (java/py/dotnot already support this) then they get written otherwise not.

Reply:
So basically, I don't want to enforce this by default. The reason is that basically right now in most of the cases user will have to use a different receiver for receiving logs like file, fluent-forward etc. That's the reason I am keeping this commented.

Reply:
Leaving it un-commented is not enforcing. For example someone might still be using jaeger receiver but that doesn't mean enabling OTLP traces is enforcing right? If they don't send logs over OTLP then it's nop. I agree people will preferably go with their existing transports mechanism but if they want to use OTLP they shouldn't have to uncomment this and it should just accept with defaults.

Reply:
Yeah, that makes sense. Let me uncomment it.